src/rpc/mod.rs

```diff
@@ -0,0 +1,37 @@
+//! Provides a rpc protocol as well as a client for the protocol
```

Comment:
please use `src/rpc.rs` as file name

Reply:
I like mod.rs, but ok...

Reply:
I don't care, but it's what we settled everywhere else on